packages/eslint-plugin-pf-codemods/lib/rules/v5/resizeObserver-function-param.js

```diff
@@ -0,0 +1,53 @@
+const { getPackageImports } = require("../../helpers");
+
+// https://github.com/patternfly/patternfly-react/pull/8324
+module.exports = {
+  meta: { hasSuggestions: true },
+  create: function (context) {
+    const imports = getPackageImports(context, "@patternfly/react-core").filter(
+      (specifier) => specifier.imported.name === "getResizeObserver"
+    );
+
+    return imports.length == 0
+      ? {}
+      : {
+          CallExpression(node) {
+            if (
+              node.callee.name === "getResizeObserver" &&
+              node.arguments.length !== 3
+            ) {
+              context.report({
+                node,
+                message: `A third parameter, useRequestAnimationFrame, has been added to the getResizeObserver function.`,
+                suggest: [
+                  {
+                    desc: "Pass in a value of false when the callback passed in is debounced. This maintains the previous functionality.",
+                    fix: function (fixer) {
+                      if (node.arguments.length !== 3) {
```

Comment:
I don't think it would get to this point with this being false, unless I am missing something.